app/uk/gov/hmrc/customs/declaration/xml/MdgPayloadDecorator.scala

```diff
@@ -35,11 +35,18 @@ class MdgPayloadDecorator() {
         <!--type: regimeType-->
         <v1:regime>CDS</v1:regime>
         <v1:receiptDate>{ dateTime.toString(ISODateTimeFormat.dateTimeNoMillis) }</v1:receiptDate>
-        <v1:clientID>{clientId}</v1:clientID>
+        <v1:clientID>{asfr.fieldsId}</v1:clientID>
         <v1:conversationID>{vpr.conversationId.uuid}</v1:conversationID>
-        { vpr.authorisedAs match {
+        {val as = vpr.authorisedAs
+      as match {
             case Csp(badgeId, _) => <v1:badgeIdentifier>{badgeId.value}</v1:badgeIdentifier>
-            case _ => NodeSeq.Empty
+            case NonCsp(eori, _) =>
+              <v1:authenticatedPartyID>{eori.value}</v1:authenticatedPartyID> // originatingPartyID is only required for CSPs
+            case CspWithEori(badgeId, eori, _) =>
+              <v1:badgeIdentifier>{badgeId.value}</v1:badgeIdentifier>
+              <v1:originatingPartyID>{eori.value}</v1:originatingPartyID>
```

Comment:
question - are we sure `asfr.fields.authenticatedEori` will exists?

Reply:
yes - I check it before hand: https://github.com/hmrc/customs-declarations/pull/203/files#diff-4ec69ceb56f18e0df51ecc2ec0655524R135